src/researchhub_document/related_models/researchhub_post_model.py

```diff
@@ -114,6 +114,13 @@ class ResearchhubPost(AbstractGenericReactionModel):
         related_query_name='posts'
     )
     slug = models.SlugField(max_length=1024)
+    doi = models.CharField(
+        max_length=255,
```

Comment:
Will DOIs ever get that long?

Reply:
They shouldn't since we control how long the suffix is, but this matches the paper model's DOI field.